WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -147,15 +136,7 @@ private extension OrderTableViewCell {
 
 private extension OrderTableViewCell {
     func configureBackground() {
-        var backgroundConfiguration: UIBackgroundConfiguration
-
-        if #available(iOS 16.0, *) {
-            backgroundConfiguration = defaultBackgroundConfiguration()
-        } else {
-            backgroundConfiguration = UIBackgroundConfiguration.listPlainCell()
-        }
-        backgroundConfiguration.backgroundColor = .listForeground(modal: false)
-        self.backgroundConfiguration = backgroundConfiguration
+        applyDefaultBackgroundStyle()
```

Comment:
Isn't this a move back to the old style? Should it be
```suggestion
        configureDefaultBackgrountConfiguration()
```

Reply:
Oh I see now I made some copy paste mistakes :D